### DISCUSSION TITLE
File Explorer crashes on newly updated ExplorerPatcher 22621.2428.59.1 dated 10-nov-2023

### DISCUSSION THREAD

**Discussion by boonduck** [valinet/ExplorerPatcher#2420] [General]
I just updated ExplorerPatcher to latest version 22621.2428.59.1 dated 10-nov-2023. 
Right after restarting, I fired up file Explorer and browsed to the download folder to locate a PDF.
I rightclicked the PDF and it took 3-5 seconds before Windows responded, where it usually takes only half a second to respond.
This time not with the contextmenu but with a crash and restart of Explorer, showing the desktop again.
After analysing the issue, I started to suspect ExplorerPatcher and uninstalled it in the usual way.
Without even rebooting I fired up File Explorer again and repeated rightclicking the PDF with now bringing up the contextmenu as expected.
Can you please solve this? Let me know if you need more information.

My config:
Windows version: Windows 11 version 22H2 (OS build 22621.2428)
ExplorerPatcher version: 22621.2428.59.1
No antivirus software installed other then the default Windows 11 one.

Thanks,
--boonduck

**Comment by pyrates999** [comment]
Please include what settings you changed in explorer patcher properties application.

**Comment by EdwardD808** [comment]
I didn't see any rules about adding to a discussion, but I have the same version.  Explorer crashes with every right click.

**Comment by pyrates999** [comment]
Duplicate of: https://github.com/valinet/ExplorerPatcher/issues/2218

Please close.
